SQLite/Extensions/FTS4.swift

```diff
@@ -138,7 +138,7 @@ extension Tokenizer : CustomStringConvertible {
 
 }
 
-extension Connection {
+extension DirectConnection {
 
     public func registerTokenizer(submoduleName: String, next: String -> (String, Range<String.Index>)?) throws {
```

Comment:
Can I use FTS with a connection pool? Because this extension is on DirectConnection, I'm inclined to think no.